index.js

```diff
@@ -103,4 +104,9 @@ app.on('before-quit', () => {
 	if (!mainWindow.isFullScreen()) {
 		storage.set('lastWindowState', mainWindow.getBounds());
 	}
+
+	// save the system's theme to determine if it changed on next startup.
+	if (process.platform === 'darwin') {
+		storage.set('lastSystemTheme', osxAppearance.isDarkMode());
```

Comment:
Why do we care what the last system theme is? Why wouldn't `anatine` keep the state specific to this app and switch based on system preferences? The system darkMode just affects the dock and the menu bar, not BG colors and stuff for apps.